python/gui/MaaS/static/maas/js/components/createDatasetForm.js

```diff
@@ -0,0 +1,42 @@
+class CreateDatasetForm {
+    constructor(parentDivId) {
+        this.parentDivId = parentDivId;
+        this.formElementId = this.parentDivId + "-form";
+        this.formContentDivId = this.formElementId + "-div-universal-inputs";
+        this.dynamicVarsDivId = this.formElementId + "-div-dynamic-inputs";
+
+    }
+
+    updateFormatChange(selection) {
+        let dy_div = document.getElementById(this.dynamicVarsDivId);
+        //let initial_length = dy_div.childNodes.length;
+        dy_div.childNodes.forEach(child => child.remove());
+
+        let addUploadSelection = false;
+        if (selection == "NETCDF_FORCING_CANONICAL") {
```

Comment:
As also noted in #198, in the future all format selections likely will have a file upload selection. Just leaving this here to relate this.